diff --git a/QuoteGeneration/pccs/services/pckcrlService.js b/QuoteGeneration/pccs/services/pckcrlService.js
index c410c26..a2511e5 100644
--- a/QuoteGeneration/pccs/services/pckcrlService.js
+++ b/QuoteGeneration/pccs/services/pckcrlService.js
@@ -45,7 +45,7 @@ export async function getPckCrl(ca, encoding) {
   }
 
   if (!encoding || encoding.toUpperCase() != 'DER') {
-    result['pckcrl'] = Buffer.from(result['pckcrl'], 'utf8').toString('hex');
+    // result['pckcrl'] = Buffer.from(result['pckcrl'], 'utf8').toString('hex');
   }
 
   return result;
diff --git a/QuoteGeneration/pccs/services/rootcacrlService.js b/QuoteGeneration/pccs/services/rootcacrlService.js
index 18f81d1..b28c629 100644
--- a/QuoteGeneration/pccs/services/rootcacrlService.js
+++ b/QuoteGeneration/pccs/services/rootcacrlService.js
@@ -48,7 +48,7 @@ export async function getRootCaCrl() {
   }
 
   // To keep backward compatibility. 
-  crl = Buffer.from(crl, 'utf8').toString('hex');
+  // crl = Buffer.from(crl, 'utf8').toString('hex');
   
   return crl;
 }
